.circleci/config.yml

```diff
@@ -658,9 +658,9 @@ jobs:
           command: rustc --version; cargo --version; rustup --version
       - restore_cache:
           keys:
-            - cargocache-v2-contract_replier-rust:1.81-{{ checksum "Cargo.lock" }}
+            - cargocache-v2-contract_replier-rust:1.82-{{ checksum "Cargo.lock" }}
       - check_contract:
-          min_version: "1.5"
+          min_version: "3.0"
```

Comment:
Had to bump the min_version here because 1.82 compiles contracts with reference-types, which are only supported starting in 3.0